docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -0,0 +1,333 @@
+by running the provided script:
+
+```bash
+cd docker-images/OracleFMWInfrastructure/samples/12213-patch-fmw-for-k8s
+./build.sh
+```
+
+This will produce an image named `oracle/fmw-infrastructure:12213-update-k8s`.
+
+All samples and instructions reference the pre-built and already patched image, `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.3`. When building your own image, you will need to rename `oracle/fmw-infrastructure:12213-update-k8s` to `container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.3`.
+
+```
+$ docker tag oracle/fmw-infrastructure:12213-update-k8s container-registry.oracle.com/middleware/fmw_infrastructure:12.2.1.3
+```
+
+After cloning the repository and downloading the installer from Oracle Technology Network
+or e-delivery, you create your image by running the provided script:
+
+```bash
+cd docker-images/OracleFMWInfrastructure/dockerfiles
+./buildDockerImage.sh -v 12.2.1.3 -s
+```
+
+The image produced will be named `middleware/soasuite/oracle/soasuite:12.2.1.3`.
+
+The Oracle SOA Suite image created through the above step needs to be retagged
+from `middleware/soasuite/oracle/soasuite:12.2.1.3` to `oracle/soa:12.2.1.3` before continuing with the next steps.
+
+```bash
+$ docker tag middleware/soasuite/oracle/soasuite:12.2.1.3 oracle/soa:12.2.1.3
+```
+
+You can use this image to run the Repository Creation Utility and to run your domain using the “domain on a persistent volume” model.
+
+Before creating a domain, you will need to set up the necessary schemas in your database.
+
+#### Configuring access to your database
+
+SOA Suite domains require a database with the necessary schemas installed in them.
+A utility called Repository Creation Utility (RCU) is provided which allows you to create
+those schemas.  You must set up the database before you create your domain.
+There are no additional requirements added by running SOA in Kubernetes; the
+same existing requirements apply.
+
+For development or test environments, you may use the container-based database,
+whereas for production environments, it is recommended that you use the standalone database.
+
+The following documentation and samples are for creating a container-based database on a persistent volume.
+
+##### Running the database inside Kubernetes
+
+Follow these instructions for a basic database setup inside Kubernetes that uses PV (persistent volume) and PVC (persistent volume claim) to persist the data. For more details about database setup and configuration, refer this [page](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/fmw-infra/#running-the-database-inside-kubernetes).
+
+Pull the database image:
+
+```bash
+$ docker pull container-registry.oracle.com/database/enterprise:12.2.0.1
+$ docker tag  container-registry.oracle.com/database/enterprise:12.2.0.1  oracle/database:12.2.0.1
+```
+Create the PV and PVC for the database:
+
+```bash
```

Comment:
Please remove lines from 164 to 176. And add the below content in its place:

The user needs to create the Kubernetes persistent volume (PV) where the database will be hosted, and the Kubernetes persistent volume claim (PVC) for the db.
To create PV/PVC run the create-pv-pvc.sh script located at kubernetes/samples/scripts/create-weblogic-domain-pv-pvc. Please follow the instructions in the page Create sample PV and PVC (Link to https://oracle.github.io/weblogic-kubernetes-operator/samples/simple/storage/).